### PR TITLE
#8 #57 finishing the skeleton for monitoring & supervised group

### PR DESCRIPTION
**Pull request by KhangZhengNg**
Create a button on homepage that directly goes into monitoring group that an user owned.
Create another button on homepage that goes to the list of groups where an user is being supervised. Clicking on the listed item will bring user to the group page.

### CODE REVIEW

**Review by Landry333 — COMMENTED** [review]
Your pull request didn't pass the checks. Can you fix that?

**Review by refatK — CHANGES_REQUESTED** [review]
Good skeleton. Need to refactor some code to remove UI-db coupling. There's some other naming issues I have mentioned which can be fixed in refactors.

The app didn't run for me though, because of the gradle version in the project-level build.gradle. I was able to run it by changing the version from 4.10 to 4.0.2

Once I made the change the app ran and I was able to see the users that are supervising me as I had expected. I'm assuming since this is just a skeleton, the visuals will be cleaned up and removal from group will be possible. Other comments I mentioned in this PR don't necessarily have to be fixed now (since they weren't made by you and don't cause the app to not run), but we should fix them eventually.


Please make these 2 changes, and then I will approve:

- [ ] Refactor Group.java to use attribute `monitoringUserId` instead of `monitorUserId`. It should match the value in the database which is `monitoringUserId`

- [ ] Change the project level build.gradle to use `classpath 'com.android.tools.build:gradle:4.0.2'` Then check that the app runs.

**Review by refatK — APPROVED** [review]
I made the changes that were crucial (gradle version, and Group.java inconsistency), so now I approve.

File: app/src/main/java/com/example/bigowlapp/ViewAnotherUserActivity.java [hunk 1]
```diff
@@ -45,43 +45,41 @@ public void onClick(View v) {
     }
 
     @SuppressLint("SetTextI18n")
```
[the remark this answers is not in the document]
Comment (Landry333) [reply app/src/main/java/com/example/bigowlapp/ViewAnotherUserActivity.java:47]: I created String variables  and passed the variables into the setText() so fix warning

File: app/src/main/java/com/example/bigowlapp/HomeActivity.java [hunk 2]
```diff
@@ -14,13 +14,9 @@
...
 import com.example.bigowlapp.repository.UserRepository;
 import com.google.firebase.auth.FirebaseAuth;
-import com.google.firebase.auth.FirebaseUser;
-
-import java.security.PrivateKey;
 
 public class HomeActivity extends AppCompatActivity {
-  
-    Button btnLogOut, SendSmsInvitation;
+    Button btnLogOut, SendSmsInvitation, btnMonitoringGroup, btnSupervisedGroup;
```
[the remark this answers is not in the document]
Comment (Landry333) [reply app/src/main/java/com/example/bigowlapp/HomeActivity.java:19]: I modified the name and wrote it in lowercase

**Review by Landry333 — APPROVED** [review]
The required changes were made, so I approve

**Pull request merged**